src/lib/kdmp-parser-structs.h

```diff
@@ -556,8 +559,8 @@ static_assert(sizeof(EXCEPTION_RECORD64) == 0x98,
               "KDMP_PARSER_EXCEPTION_RECORD64's size looks wrong.");
 
 struct HEADER64 {
-  static const uint32_t ExpectedSignature = 0x45474150; // 'EGAP'
-  static const uint32_t ExpectedValidDump = 0x34365544; // '46UD'
+  static const inline uint32_t ExpectedSignature = 0x45474150; // 'EGAP'
+  static const inline uint32_t ExpectedValidDump = 0x34365544; // '46UD'
```

Comment:
`constexpr` doesn't work here instead of `const inline` I am guessing?

Reply:
True, I'm not sure how `constexpr` will help since we're unlikely to do any parsing at compile time right?